pkgs/applications/misc/qdirstat/default.nix

```diff
@@ -3,15 +3,15 @@
 , makeWrapper, perlPackages }:
 
 let
-  version = "1.5.90";
+  version = "1.6";
 in stdenv.mkDerivation rec {
   name = "qdirstat-${version}";
```

Comment:
use pname  + there is no need to define version outside if the attribute set is then marked as rec {}

Reply:
might need to use qt5.mkDerivation instead as other packages ?